### ISSUE TITLE
Accessablity

### ISSUE THREAD

**Issue by Heather-Herbert** [openwebgisystem/OpenWebGIS#2]
There are a number of accessibility issues with the page(s) including missing alt tags and colour contrast issues.
While, by their nature, maps fail accessibility checks in relation to inaccessible to most screen readers their are steps which can be taken to improve the site.
The site should aim to pass as many of the wave accessibility tests as it can (see http://wave.webaim.org/ )
